start-firestore

```diff
@@ -6,8 +6,14 @@ if [[ -z "${FIRESTORE_PROJECT_ID}" ]]; then
   exit 1
 fi
 
+# Check user environment variable
+if [[ -z "${FIRESTORE_PORT}" ]]; then
+  FIRESTORE_PORT=8088
+  echo "Missing FIRESTORE_PORT environment variable. Set default one to ${FIRESTORE_PORT}" >&2
+fi
+
 # Config gcloud project
 gcloud config set project ${FIRESTORE_PROJECT_ID}
 
 # Start emulator
-gcloud beta emulators firestore start --host-port=0.0.0.0:8080
+gcloud beta emulators firestore start --host-port=0.0.0.0:${FIRESTORE_PORT}
```

Comment:
Is the emulator still in beta with gcloud ?

Reply:
yes  https://cloud.google.com/sdk/gcloud/reference/beta/emulators/firestore